### ISSUE TITLE
Please open source the kernel source code of Nubia Red Magic 5G in Android R

### ISSUE THREAD

**Issue by Andieqqq** [ztemt/NX659J_Q_kernel#7]
The latest official version of the system is Android 11. This kernel version is outdated. Please update it as soon as possible to keep the product open source. Thank you.

**Comment by mrslezak** [comment]
Yeah you guys forgot the whole arch/arm64/boot/dts/vendor/* directory, it's pretty important since it includes all the functions necessary to build the kernel for your specific device.  The Q kernel for RM5S at least had qcom nubia and bindings inside so you could build it.  I would expect RM5S / RM5G code in there since you updated the ROMs to support both devices, but no one uploaded it anywhere.
